src/NexusMods.App/Listeners/NxmRpcListener.cs

```diff
@@ -39,12 +39,12 @@ private async Task ListenAsync()
         while (await _urlsBlock.OutputAvailableAsync(_cancellationSource.Token))
         {
             var item = await _urlsBlock.ReceiveAsync(_cancellationSource.Token);
-            
+
             // The following code is wrapped in an exception handler such that on possible unhandled exception,
             // the service keeps running rather than terminated.
             try
             {
-                _downloadService.AddNxmTask(item.Value);
+                _ = _downloadService.AddNxmTask(item.Value);
```

Comment:
This seems odd to me, we should either name this return variable or not assign it.

Reply:
If you have any stylistic preference here, let me know.